package.json

```
@@ -19,7 +19,7 @@
     "typecheck": "turbo run typecheck",
     "knip": "knip",
     "validate-config": "turbo run validate-config",
-    "merge-checks": "pnpm build && turbo run publint manypkg typesync knip typecheck lint validate-config check-deduped-deps are-the-types-wrong",
+    "merge-checks": "pnpm build && turbo run publint manypkg typesync knip typecheck lint validate-config are-the-types-wrong",
```

i think we can do `&& turbo run check-deduped-deps` at the end here in `merge-checks` and remove the call from the `.github/workflows/merge-checks.yml`

---

No, because deduping deleting node_modules, so it breaks the other scripts, unfortunately.

---

if we do `&& turbo run check-deduped-deps` at the end, it will run the dedupe only **after** all the other scripts have run. So it shouldn't break them.

---

Oh, 🤦‍♂️
Yeah, I misread your suggestion.

```suggestion
    "merge-checks": "pnpm build && turbo run publint manypkg typesync knip typecheck lint validate-config are-the-types-wrong && turbo run check-deduped-deps",
```